src/components/ListingDetail/ListingDetail.html

```diff
@@ -1,11 +1,48 @@
-<div>
+<div class="container">
   <div v-if="isLoading">
     Carregando detalhes do anúncio...
   </div>
   <div v-else>
+    <div class="image-container">
+      <VueSlickCarousel :="settings">
+        <div class="carousel-bg-image" v-for="media in listing.medias">
+          <img class="carousel-image" :src="media.url" alt="">
```

Comment:
Why property `alt` is empty ?
This tag is very important to SEO!! because when the image not exist the title is show

Reply:
Right, Right!
Ive forgot to write. Thanks!